drivers/i2s/i2s_mcux_sai.c

```diff
@@ -333,21 +332,19 @@ static void enable_mclk_direction(const struct device *dev, bool dir)
 	uint32_t offset = dev_cfg->mclk_pin_offset;
 	uint32_t mask = dev_cfg->mclk_pin_mask;
 	uint32_t value = 0;
+	uint32_t *gpr = (uint32_t *)DT_REG_ADDR(DT_NODELABEL(iomuxcgpr)) + offset;
```

Comment:
where is the DT node defined now?

Reply:
@gmarull this is defined in the [here](iomuxcgpr: iomuxcgpr@400ac000 {)